src/components/form/HdTextarea.vue

```diff
@@ -80,6 +90,10 @@ export default {
       type: String,
       default: '100px',
     },
+    maxlength: {
+      type: Number,
+      default: () => Number.POSITIVE_INFINITY,
```

Comment:
Couldn't you just use the value directly instead of a factory function?

```suggestion
      default: Number.POSITIVE_INFINITY,
```

Reply:
https://github.com/homeday-de/homeday-blocks/pull/815/commits/709415abb180a5a8b4e86abcf78dd008768fc645